content/english/anti-vegan/pests.md

```diff
@@ -0,0 +1,127 @@
+
+> Environmental Ethics is complex and difficult. Even assuming we can reach agreement on the definition of pest it does not automatically follow that humans *ought* to control or eradicate this species.
+
+  
+## Detail - Analytic
+
+It is difficult to interpret how the central tenet of veganism *‘avoid exploitation of and cruelty to animals'* might be applied in complex environmental considerations. 
+
+However, before we try, numerous concepts require unpacking.
+
+### 1. Veganism is moral standpoint. 
+
+
+Veganism is not a moral theory.  Veganism is a moral standpoint regarding the (mis)treatment of animals by (most) humans, either directly or by acquiescence. We can’t expect specific moral standpoints to offer complete answers to questions that are fully or partially outside their scope.  That said, there is considerable crossover between environmental and animal concerns so intuitively we expect that vegans ought to hold a position on environmental issues     
+### 2. What is a pest?
+
+Pest is commonly defined along the lines of any “small animal which damages crops or food supplies” (Collins) or “any animal or plant detrimental to human concerns” (Wikipedia).  
+
+> Speciesism is a form of discrimination based on species membership. It involves treating members of one species as morally more important than members of other species even when their interests are equivalent.  
+>  - [Wikipedia](https://en.wikipedia.org/wiki/Speciesism)
+
+Vegans will NOT accept this deginition of pest as it is 'speciesist’. That an animal’s interests conflict with human interests is not, in itself, a reason for defining the animal as a pest or for taking action against them. Where crops are concerned alternative means such as fencing, sonics, and eco-friendly organic practices are available.  
+
+## Re-defining ‘pest’.
+
+> "A thing is right when it tends to preserve the integrity, stability and beauty of the biotic community. It is wrong when it tends otherwise.”  
+> - Aldo Leopold (1949) [The Land Ethic](https://rintintin.colorado.edu/~vancecd/phil3140/Leopold.pdf) 
+
+Aldo Leopold, the father of environmental ethics, provided a non-speciesist definition of pest when he claimed “A thing is right when it tends to preserve the integrity, stability and beauty of the biotic community. It is wrong when it tends otherwise.”  If we apply this axiom a species is a pest when, regardless of human interest, it upsets the stability of, interrupts the integrity of, or leads to the destruction of the environment.  
+
+Two New Zealand specific examples:
+
+### Are wild rabbits a pest?
+
+Biosecurity NZ lists rabbits as pests because they ‘compete with livestock, cause extensive damage to farming land, cause public nuisance… 
+
+So, yes, they are annoying to (some) humans but they are not a ‘pest’. The reasons provided by Biosecurity NZ fail under Leopold’s criteria. If wild rabbits are a pest other substantive, non-speciesist, reasons must be provided.
```

Comment:
`substantive` is not a word regular English speakers understand :p

Reply:
Really? maybe 'more substantial?'

Reply:
Perfect!